src/Integration.UnitTests/Notifications/SonarQubeNotificationServiceTests.cs

```diff
@@ -110,7 +110,7 @@ public void StartAsync_Throws_ArgumentNullException_For_ProjectKey()
             await notifications.StartAsync("should-throw", null);
 
             // Assert
-            outputMock.Verify(x => x.WriteLine($"Failed to fetch notifications: test exception"), Times.Once);
+            outputMock.Verify(x => x.WriteLine("Failed to fetch notifications: {0}", "test exception"), Times.Once);
```

Comment:
Is this change just about readability or was there any problem?

Reply:
It was required since this test is using a mock. Previously the product code was calling _WriteLine(string)_, now it's calling _WriteLine(string, object[] args)_, so the test needs to verify a different overload.